### PR TITLE
fix: pass env to Node-API and remove `patchJest`

### PR DESCRIPTION
**Pull request by williamluke4**
[Engines PR](https://github.com/prisma/prisma-engines/pull/2004)

### CODE REVIEW

File: src/packages/client/helpers/patchJest.js [hunk 1]
```diff
@@ -1,19 +0,0 @@
-const resolveFrom = require('resolve-from')
```
Comment (Jolg42) [inline src/packages/client/helpers/patchJest.js:1]: 🍾

Comment (williamluke4) [reply src/packages/client/helpers/patchJest.js:1]: Nice catch ill remove it

File: src/packages/sdk/package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "format": "prettier --write .",
     "lint": "eslint --cache --fix --ext .ts .",
     "lint-ci": "eslint --ext .ts .",
-    "test": "node ./scripts/patchJest && jest",
+    "test": "jest --forceExit --verbose --clearCache",
```
Comment (Jolg42) [inline src/packages/sdk/package.json:21]: --verbose is nice!
--forceExit was not needed before, do you know why it's needed now?
--clearCache that's new? why do we need this?

**Review by Jolg42 — APPROVED** [review]
See --forceExit --clearCache question before we merge

File: src/packages/sdk/package.json (hunk 2 above)
Comment (williamluke4) [reply src/packages/sdk/package.json]: removed --clearCache

Comment (williamluke4) [reply src/packages/sdk/package.json]: I have no idea why `--forceExit` is now needed. We would need to investigate.

**Comment by janpio** [comment]
🥳

**Pull request merged**